tests/draft2020-12/unevaluatedItems.json

```diff
@@ -433,5 +433,134 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "item is evaluated in a sibling schema to unevaluatedItems",
+        "schema": {
+            "type": "array",
+            "prefixItems": [
+                {
+                    "type": "string"
+                }
+            ],
+            "unevaluatedItems": false,
+            "anyOf": [
+                {
+                    "prefixItems": [
+                        {
+                            "type": "string"
+                        }
+                    ]
+                }
+            ]
+        },
+        "tests": [
+            {
+                "description": "no extra items",
+                "data": [
+                    "test"
+                ],
+                "valid": true
+            },
+            {
+                "description": "uncle keyword evaluation is not significant",
+                "data": [
+                    "test",
+                    "test"
+                ],
+                "valid": false
+            }
+        ]
+    },
```

Comment:
This appears to be a duplicate of "unevaluatedItems with nested tuple" on line 120

Reply:
Agreed, the same duplication happened as with unevaluatedProperties, as I essentially just copied the new unevaluatedProperties tests and modified the schema and data to use arrays instead.